charts/datahub/subcharts/datahub-frontend/templates/ingress.yaml

```diff
@@ -30,8 +31,8 @@ spec:
         paths:
         {{- range .redirectPaths }}
           - path: {{ .path }}
-            {{- if eq $ingressApiVersion "networking.k8s.io/v1" }}
-            pathType: ImplementationSpecific
+            pathType: {{ $.Values.ingress.pathType | default "Prefix" | quote }}
```

Comment:
Can we make ImplementationSpecific as default so that ingress remains the same if this field is not set?